packages/sources/react/src/react-app-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```

Comment:
is that file useful ?

Reply:
Good question, I will check!

Reply:
This file references the types of react-scripts, and helps with things like allowing for SVG imports :

[source](https://www.sitepoint.com/react-with-typescript-best-practices/#:~:text=json%20is%20the%20TypeScript%20configuration,like%20allowing%20for%20SVG%20imports.)

Reply:
Oh okay thanks @NicolasBreunin, we will keep it so.